packages/ProgressIndicators/src/ProgressBar/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+/** Props for ProgressBar */
+interface ProgressBarProps {
+  anime: boolean;
+  decimalPoints: number;
+  height: string;
+  min: number;
+  max: number;
+  value: number;
+  lineColor: string;
+  lineColorThresholds?: { [key: string]: number } | undefined;
+  showLabel: boolean;
+  stripped: boolean;
+}
+
+/** default props for ProgressBar */
+const defaultProgressBarProps: Partial<ProgressBarProps> = {
+  anime: false,
+  decimalPoints: 0,
+  height: '10px',
+  lineColor: '#0000FF',
+  lineColorThresholds: undefined,
+  max: 100,
+  min: 0,
+  showLabel: false,
+  stripped: false,
+  value: 0
+};
+
+/** Displays configurable progress bar
+ * lineColor prop when you don't require different line colors
```

Comment:
This style of docstring more naturally belongs next to the props inside `ProgressBarProps` e.g. something like:

```ts
...
lineColor: string /** description goes here */;
...
```
Additionally, I would do this for all the other props.